cli/packages.go

```diff
@@ -178,9 +178,11 @@ func PackageUploadAction(c *cli.Context) error {
 		return err
 	}
 
-	_, err = convertPackageTypeToProto(c.String(packageFlagType))
-	if err != nil {
-		return err
+	// Default to "unspecified" if no package type is provided
```

Comment:
I'm not sure this change will solve the problem. The first convertPackageTypeToProto is to ensure that there is a valid package type before passing it along to the upload code. TFLite, Tensorflow, PyTorch, etc.. are all model frameworks not package types. 

From the thread it says "the CLI doesn't allow me to specify what kind of model it is, and it incorrectly put it as a TFLite file, rather than Tensorflow", meaning somewhere we are setting the default model _framework_ to be TFLite & furthermore we should allow people to specify the model _framework_ in the upload command. Here's an [example](https://github.com/viamrobotics/rdk/blob/9bcc9592ba1a6f5efe96dd425337d43b715fe65c/cli/ml_training.go#L486) of how we set the model framework for a different resource (ML training scripts instead of ML models). So as part of this ticket, you would need to figure out: how the framework gets set for ML models and how to amend package upload CLI to take in the model framework.

Reply:
yeah it's definitely not the solution. I realized it when I started testing it. Thanks for the info. I'll update this PR accordingly. Leaving it in draft mode for now